db/migrate/20170728202534_add_gemnasium_field_to_repositories.rb

```diff
@@ -0,0 +1,5 @@
+class AddGemnasiumFieldToRepositories < ActiveRecord::Migration[5.1]
```

Comment:
Instead of adding a new migration to change Repositories, how about editing the initial Repository migration and schema directly -- in the interest of cleaner migration history, at least until the 1.0 release?